src/Symfony/Component/HttpKernel/composer.json

```diff
@@ -19,7 +19,7 @@
         "php": "^5.5.9|>=7.0.8",
         "symfony/event-dispatcher": "~2.8|~3.0|~4.0",
         "symfony/http-foundation": "~3.4.12|~4.0.12|^4.1.1",
-        "symfony/debug": "~2.8|~3.0|~4.0",
+        "symfony/debug": "~3.3|~4.0",
```

Comment:
Why `~3.3` and not `~3.2`?

Reply:
should be `^3.3.3|~4.0` then

Reply:
I just realized that. it's strange that a public property was added in a minor version.